README.md

```diff
@@ -33,7 +33,7 @@ Rustls is a TLS library that aims to provide a good level of cryptographic secur
 requires no configuration to achieve that security, and provides no unsafe features or
 obsolete cryptography by default.
 
-## Current features
+## Current features (with default crate features)
```

Comment:
I'm wondering if we can avoid saying "features" twice here, where one time we mean Cargo features and another more generically the functionality Rustls has:

```suggestion
## Current functionality (with default crate features)
```